### DISCUSSION TITLE
midi-ble?

### DISCUSSION THREAD

**Discussion by fabiendostie** [marcel-licence/esp32_usb_midi#10] [Q&A] [answered]
are you planning on integrating one of the midi-ble libraries to your projects? it could be a great addition i think!

**Comment by marcel-licence** [comment, marked as answer]
Yes I think I will give it a try. I may affect the performance. I will see

**Reply by fabiendostie** [reply]
It would be great!!
Keep me posted on this... as i'm still struggling with the midi usb on the audio kit v2.2.

**Reply by fabiendostie** [reply]
did you end up integrating it?

**Reply by marcel-licence** [reply]
It will take some days because I have some other things on my queue

**Comment by fabiendostie** [comment]
No problem you take your time.

Fabien Dostie
Monteur Online / Coloriste Numérique (AQTIS)
Datawrangler-DIT-Video Assist
(AQTIS/IATSE)

**Reply by marcel-licence** [reply]
I did some testing. It looks like putting the `BLEMIDI_CREATE_INSTANCE` into the code slows it completely down. I am not sure what it really changes to the code execution

**Reply by marcel-licence** [reply]
In addition to that when removing `btStop();` from Audio_Setup() the synthesizer does not work any more.

**Reply by marcel-licence** [reply]
Without the synth I think it would work without any problems =) I prepared a little project some weeks ago
